app/scripts/views/sign_up.js

```diff
@@ -83,7 +84,7 @@ define(function (require, exports, module) {
       return coppaView.render()
         .then(() => {
           this.trackChildView(coppaView);
-          coppaView.on('submit', this.validateAndSubmit.bind(this));
+          coppaView.on('submit', () => this.trigger('submit'));
```

Comment:
This seemed very strange to me, I had to test this to ensure it worked as expected.